libethcore/Farm.h

```diff
@@ -396,10 +396,12 @@ class Farm: public FarmFace
 		return stream.str();
 	}
 
-	void set_pool_addresses(string primaryUrl, string primaryPort, string failoverUrl, string failoverPort) {
-		m_pool_addresses = primaryUrl + ":" + primaryPort;
+	void set_pool_addresses(string primaryUrl, unsigned primaryPort, string failoverUrl, unsigned failoverPort) {
```

Comment:
If you like, you can change that as-well. ``m_pool_addresses`` is used only by the api and api is based on claymore. In claymores api "spec", only the active pool is displayed, so it makes no sense to show multiple pools there.   

Also PoolManager calls ``set_pool_addresses`` with only the active connection and leaves the failover empty.

In my opinion it makes sense to only display the CURRENT not all addresses.

Reply:
Makes sense.

Reply:
Do you care if I squash?